packages/jupyterlab-chat/src/widget.tsx

```diff
@@ -205,9 +205,9 @@ export class ChatPanel extends SidePanel {
    */
   updateChatList = async (): Promise<void> => {
     const extension = chatFileType.extensions[0];
-    this._drive
+    this._contentsManager
       .get(this._defaultDirectory)
-      .then(contentModel => {
+      .then((contentModel: { content: any[] }) => {
```

Comment:
```suggestion
      .then((contentModel: Contents.IModel) => {
```

Can't we use the expected type ?